pisek/jobs/parts/completeness_check.py

```diff
@@ -71,8 +71,8 @@ def _check_cms_judge(self) -> None:
             for judge_out in self._get_judge_outs():
                 with open(judge_out.path) as f:
                     lines = f.read().rstrip().split("\n")
```

Comment:
Won't `split` with `sep` specified always return a non-empty list?

Reply:
First!

Yeah this behaviour is really weird...

Reply:
> Yeah this behaviour is really weird...

Not really...

Reply:
Nit: It might be nice to have two separate messages for > 1 and < 1, but you'll probably want to look at the file anyway.